lib/ansible/modules/net_tools/ldap/ldap_attr.py

```diff
@@ -279,8 +278,7 @@ def main():
             try:
                 ldap.connection.modify_s(ldap.dn, modlist)
             except Exception as e:
-                module.fail_json(msg="Attribute action failed.", details=to_native(e),
-                                 exception=traceback.format_exc())
+                module.fail_json(msg="Attribute action failed.", details=to_native(e))
```

Comment:
What is the effect of removing the exception?

Reply:
The exception is added by Ansible by default, so we don't need them individually in each module. So no effect.